pavelib/assets.py

```diff
@@ -184,6 +213,7 @@ def compile_coffeescript(*files):
 @task
 @no_help
 @cmdopts([
+    ('system=', 's', 'System to act on'),
```

Comment:
Good point. This was a cut-and-paste from another Paver command, so they all seem to have that string. Also note that this command has ``@no_help`` so I think the string is ignored. I'll update it just in case.